mm2src/common/privkey.rs

```diff
@@ -93,7 +93,7 @@ pub fn key_pair_from_seed(seed: &str) -> PrivKeyResult<KeyPair> {
     Ok(pair)
 }
 
-pub fn key_pair_from_secret(secret: [u8; 32]) -> PrivKeyResult<KeyPair> {
+pub fn key_pair_from_secret(secret: &[u8]) -> PrivKeyResult<KeyPair> {
```

Comment:
It already returns `Result` because not every 32 bytes sequence is a valid secp256k1 so renaming it to try doesn't really seem reasonable to me. But I think it's worth adding a slice length check because it will extend or cut any bytes sequence to *some* H256.

Reply:
Sounds good!